R/makeDisplay.R

```diff
@@ -125,8 +125,11 @@ makeDisplay <- function(
   }
 
   if(verbose) message("* Validating 'panelFn'...")
-  panelEx <- kvApply(kvExample(data), panelFn)$value
-
+  if(!is.null(params) && class(params) == "list"){
+  	environment(panelFn) = list2env(params)
+  	environment(cogFn) = list2env(params)
+  }
+  panelEx =  kvApply(kvExample(data), panelFn)$value
```

Comment:
Not to be nitpicky, but to stick with consistent style, could you change it to `panelEx <-`